cypress/integration/register.spec.js

```diff
@@ -6,6 +6,7 @@ describe('register', function() {
   const newUser = {
     email: faker.internet.email(),
 
+    existingEmail: 'kylemh.email12@gmail.com',
```

Comment:
This doesn't really belong as part of `newUser` could you declare it as it's own variable?

`const existingEmail = 'kylemh.email12@gmail.com';`

Reply:
Good catch! Im not sure why i declared it there. Fixed!